servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -2730,6 +2730,24 @@ uint32_t RenderForwardMobile::get_max_elements() const {
 	return 256;
 }
 
+bool RenderForwardMobile::get_material_has_transparency(RID p_material_rid) const {
+	SceneShaderForwardMobile::MaterialData *material = dynamic_cast<SceneShaderForwardMobile::MaterialData *>(RendererRD::MaterialStorage::get_singleton()->material_get_data(p_material_rid, RendererRD::MaterialStorage::SHADER_TYPE_3D));
+	// Can't found the SceneMaterialData corresponding to p_material_rid
+	if (!material)
+		return false;
+
+	bool has_read_screen_alpha = material->shader_data->uses_screen_texture || material->shader_data->uses_depth_texture || material->shader_data->uses_normal_texture;
+	bool has_base_alpha = material->shader_data->uses_alpha && (!material->shader_data->uses_alpha_clip || material->shader_data->uses_alpha_antialiasing);
+	bool has_blend_alpha = material->shader_data->uses_blend_alpha;
+	bool has_alpha = has_base_alpha || has_blend_alpha || has_read_screen_alpha;
+
+	if (has_alpha || material->shader_data->depth_draw == SceneShaderForwardMobile::ShaderData::DEPTH_DRAW_DISABLED || material->shader_data->depth_test == SceneShaderForwardMobile::ShaderData::DEPTH_TEST_DISABLED) {
+		//material is only meant for alpha pass
+		return true;
+	}
+	return false;
+}
```

Comment:
Based on : https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp#L2368
I could have used my new method in place of the original code, but I didn't want to add regressing in a bugfix.